protocol/contracts/profit/src/state/open_ica.rs

```diff
@@ -6,10 +6,10 @@ use dex::{Account, ConnectionParams, DexConnectable, IcaConnectee};
 
 use crate::{error::ContractError, msg::ConfigResponse, result::ContractResult};
 
-use super::{idle::Idle, Config, ConfigManagement, IcaConnector, SetupDexHandler, State};
+use super::{idle::Idle, Config, ConfigManagement, IcaConnector, State};
 
 #[derive(Debug, Serialize, Deserialize)]
-pub(super) struct OpenIca {
+pub(crate) struct OpenIca {
```

Comment:
why do we need to relax the visibility?

Reply:
because of the error that the type is more private than the item state::State::start.

Reply:
unneeded due to the way we start a State